explore/visualization/README.md

```diff
@@ -0,0 +1,9 @@
+# A directory for creating Vega based visualizations
+
+In order to run these sample files, you should first start up a simple HTTP server such as:
+
+```sh
+python -m SimpleHTTPServer 8000
+```
+
+Then navigate to that instance (localhost:8000) and click on the file that you wish to view.

```

Comment:
Replacing `localhost:8000` with `http://localhost:8000/` will make the link clickable -- just worried that some ML devs will be confused by what localhost is.